cmd/search_log.go

```diff
@@ -341,15 +342,25 @@ func latencyAggregateFn(cmd *cobra.Command, args []string) {
 	})
 	log.Printf("%d worst queries:\n", worstQueriesPrintCnt)
 	formatedHeaders := make([]string, 0)
-	for _, r := range records[0] {
-		fr := strings.Replace(r, ".", " ", -1)
-		formatedHeaders = append(formatedHeaders, fr)
+	if len(records)==0 {
+		fr := ""
+		formatedHeaders = append(formatedHeaders, fr)	
+	
+	}	else {
```

Comment:
remove spaces and empty lines (should be done automatically with Go extension in VS Code)